examples/src/ui.rs

```diff
@@ -39,6 +39,28 @@ impl RenderNotifier for Notifier {
     }
 }
 
+pub trait HandyDandyRectBuilder {
```

Comment:
Just make intrinsic methods on `LayoutPrimitiveInfo`? As far as I know you never use the `by` method too. At the very least, remove `HandyDandy`.